databricks_cli/configure/provider.py

```diff
@@ -258,34 +261,56 @@ def get_config(self):
         username = _get_option_if_exists(raw_config, self.profile, USERNAME)
         password = _get_option_if_exists(raw_config, self.profile, PASSWORD)
         token = _get_option_if_exists(raw_config, self.profile, TOKEN)
+        refresh_token = _get_option_if_exists(raw_config, self.profile, REFRESH_TOKEN)
         insecure = _get_option_if_exists(raw_config, self.profile, INSECURE)
         jobs_api_version = _get_option_if_exists(raw_config, self.profile, JOBS_API_VERSION)
-        config = DatabricksConfig(host, username, password, token, insecure, jobs_api_version)
+        config = DatabricksConfig(host, username, password, token,
+                                  refresh_token, insecure, jobs_api_version)
         if config.is_valid:
             return config
         return None
 
 
 class DatabricksConfig(object):
-    def __init__(self, host, username, password, token, insecure, jobs_api_version=None):  # noqa
+    def __init__(self, host, username, password, token,
+                 refresh_token=None, insecure=None, jobs_api_version=None):  # noqa
```

Comment:
Is there an exhaustive way of knowing we have changed all the calls to init or `from_token` in the code-base if required? I am concerned if we have not used the PEP standards and specified optional parameters without giving their name. E.g. 
```
def optional_params(one, two=None):
  print(one)
  print(two)

optional_params("required_param", "first optional param")
```